app/Jobs/Notifications/BeatmapsetDiscussionPostNotification.php

```diff
@@ -16,6 +16,15 @@ abstract class BeatmapsetDiscussionPostNotification extends BroadcastNotificatio
 
     protected $beatmapsetDiscussionPost;
 
+    public static function getBaseKey(Notification $notification): string
+    {
+        $category = Notification::nameToCategory($notification->name);
+
+        return $category === 'beatmapset_discussion'
+            ? "{$notification->notifiable_type}.{$category}.beatmapset_discussion_post_new"
```

Comment:
it says "post_new" but it also includes discussion lock/unlock :thinking: 

just generic `_` maybe?